common/app/views/fragments/containers/facia_cards/container.scala.html

```diff
@@ -42,7 +42,7 @@
             @if(FlagshipFrontContainer.displayFlagshipContainer()) {
                 <section id="@componentName"
                     class="@GetClasses.forContainerDefinition(containerDefinition)"
-                    data-link-name="container-@{containerDefinition.index + 1} | @componentName"
+                    data-link-name="container-@{containerDefinition.index} | @componentName"
```

Comment:
We want to let `palette-style-do-no-delete` have index `0`, as it is stripped away in DCR.